tests/fixtures/issues/issue-523.php

```diff
@@ -1079,7 +1079,7 @@
             'Platform_Bits' => '32',
             'Platform_Maker' => 'Apple Inc',
             'Alpha' => false,
-            'Beta' => true,
+            'Beta' => false,
```

Comment:
Opera mini is marked as a beta, but when I moved the iOS patterns to their own file, I removed that property as there doesn't seem to be any indication that Opera Mini is beta on iOS: https://itunes.apple.com/app/id363729560?mt=8&pt=341230&ct=ose_google_via_opera_com_mobile_mini_iphone_top